src/OpenSpaceToolkit/Astrodynamics/Estimator/OrbitDeterminationSolver.cpp

```diff
@@ -95,9 +95,9 @@ OrbitDeterminationSolver::Analysis OrbitDeterminationSolver::estimate(
     const State initialGuessStateInEstimationFrame = anInitialGuessState.inFrame(estimationFrameSPtr_);
 
     const Array<State> observationStatesInEstimationFrame = anObservationStateArray.map<State>(
-        [estimationFrameSPtr = estimationFrameSPtr_](const State& aState) -> State
+        [this](const State& aState) -> State
```

Comment:
Why make it copy the entire object when we just need 1 shared pointer?

Reply:
Its not copying the entire object, `this` is a pointer, so it just captures the pointer by value, which is effectively the same as capturing the `estimationFrameSPtr` by value

Reply:
```suggestion
        [estimationFrameSPtr_](const State& aState) -> State
```
I guess we could also just do this
or 
```suggestion
        [&estimationFrameSPtr_](const State& aState) -> State
```
although we would have to check if the latter implementation is support in cpp versions before cpp20, cause I know [&this] capture is not supported before cpp20